vxmessenger/transport.py

```diff
@@ -175,6 +183,16 @@ class MessengerTransport(HttpRpcTransport):
     def setup_transport(self):
         yield super(MessengerTransport, self).setup_transport()
         self.pool = HTTPConnectionPool(self.clock, persistent=False)
+        self.BATCH_API_URL = self.config.get('outbound_url',
+                                             'https://graph.facebook.com')
+
+        static_config = self.get_static_config()
+        self.batch_size = static_config.request_batch_size
+        self.batch_time = static_config.request_batch_wait_time
+
+        self.redis = yield TxRedisManager.from_config(
+            static_config.redis_manager)
```

Comment:
If it's not provided in the config, it defaults to an empty dict (in which case the redis manager just starts with its default settings). I can refactor a bit so that if the transport can't find a redis connection it falls back to the ordinary request setup, but isn't redis a Junebug requirement?